### PR TITLE
feat: config url option

### PR DESCRIPTION
**Pull request by simboonlong**
- replace `config-type` option with `config-template-url` option

### CODE REVIEW

**Review by isaackps — COMMENTED** [review]
i have no issue in this but i would think there would best be a place to get the url for us to use if need be.

File: src/implementations/config/types.ts [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+import { Wallet } from "ethers";
 
-type WalletEncrypted = {
-  type: string;
+type WalletEncryptedJson = {
+  type: "ENCRYPTED_JSON";
   encryptedJson: string;
 };
 
+type WalletOptions = Wallet | WalletEncryptedJson;
```
Comment (isaackps) [inline src/implementations/config/types.ts:9]: we dont use wallet type anymore right? are u catering for older versions? im just thinking we should stick to the newer version, walletEncryptedJson

Comment (simboonlong) [reply src/implementations/config/types.ts]: ok

**Review by isaackps — APPROVED** [review]
LGTM!

**Pull request merged**